organizations/forms.py

```diff
@@ -0,0 +1,72 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.forms.models import inlineformset_factory, modelform_factory
+from django.utils.translation import ugettext as _
+
+from organizations.models import Organization
+
+
+OrganizationForm = modelform_factory(Organization, fields=('name',))
+
+
+class NewUserForm(forms.ModelForm):
+    email = forms.EmailField()
+
+    def clean_email(self):
+        User = get_user_model()
+        # TODO: handle non-unique email
+        try:
+            self.instance.user = User.objects.get(
+                email=self.cleaned_data['email'])
+        except User.DoesNotExist:
```

Comment:
I actually need to do the `self.instance.user = bla bla` assignment over there, because the form doesn't do it